README.md

```diff
@@ -1,9 +1,9 @@
-## ByteString: Fast, Packed Strings of Bytes
+# ByteString: Fast, Packed Strings of Bytes
 
-[![Build Status](https://secure.travis-ci.org/haskell/bytestring.png?branch=master)](http://travis-ci.org/haskell/bytestring)
+[![Build Status](https://secure.travis-ci.org/haskell/bytestring.svg?branch=master)](http://travis-ci.org/haskell/bytestring) [![Hackage](http://img.shields.io/hackage/v/bytestring.svg)](https://hackage.haskell.org/package/bytestring) [![Hackage CI](https://matrix.hackage.haskell.org/api/v2/packages/bytestring/badge)](https://matrix.hackage.haskell.org/package/bytestring) [![Stackage LTS](http://stackage.org/package/bytestring/badge/lts)](http://stackage.org/lts/package/bytestring) [![Stackage Nightly](http://stackage.org/package/bytestring/badge/nightly)](http://stackage.org/nightly/package/bytestring)
```

Comment:
Can we please remove the Stackage badges? They serve no benefit for a GHC bundled package such as `bytestring` and only clutter the README with useless information.

Reply:
I actually find such information useful. Otherwise I have to keep in mind 1) which version of GHC is available from Stackage today, 2) the exact mapping between GHC (including minor releases) and `bytestring` versions. It is quite convenient to see at a glance how far Stackage is lagging back. For example, a user of Stack may very well consider adding `bytestring-0.11` to `extra-deps`, if he/she learns that Stackage is one major version behind.